marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -241,6 +247,10 @@ function buildAndCacheSelectedPropertyMetadata(selectedPropertyName, selectedPro
   return finalMetadataProperty;
 }
 
+function getEntityInstance(docUri) {
+  return es.instanceJsonFromDocument(cts.doc(docUri)).toArray();
```

Comment:
How is this returning an array? http://docs.marklogic.com/es.instanceJsonFromDocument says it should return an objectNode. The name "getEntityInstance" also implies that a single thing is being returned, not an array. 

Perhaps you just need to call fn.head on the result of es.instanceJsonFromDocument, as shown in the docs page for that function?